From: Benjamin Berg <bberg@redhat.com>
Date: Fri, 20 May 2022 14:35:23 +0200
Subject: test: Allow daemon to take 5 seconds for shutdown

It seems like shutdown can take more than 2 seconds on some slow
systems. So increase to 5 seconds which hopefully is long enough.

Closes: #188
(cherry picked from commit f1c00e50c2033c206f585398fcc2ea329f4494fa)
---
 src/linux/integration-test.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/linux/integration-test.py b/src/linux/integration-test.py
index 4319a15..8c0ae2a 100755
--- a/src/linux/integration-test.py
+++ b/src/linux/integration-test.py
@@ -231,7 +231,7 @@ class Tests(dbusmock.DBusTestCase):
             except OSError:
                 pass
             try:
-                self.assertEqual(self.daemon.wait(timeout=2.0), 0)
+                self.assertEqual(self.daemon.wait(timeout=5.0), 0)
             except TimeoutError:
                 try:
                     self.daemon.kill()
